cim1/cim1-converter/src/main/java/com/powsybl/cim1/converter/CIM1ConverterConfig.java

```diff
@@ -27,7 +27,7 @@
     CIM1ConverterConfig(boolean invertVoltageStepIncrementOutOfPhase, Country defaultCountry,
                         List<String> substationIdExcludedFromMapping, CIM1NamingStrategyFactory namingStrategyFactory) {
         this.invertVoltageStepIncrementOutOfPhase = invertVoltageStepIncrementOutOfPhase;
-        this.defaultCountry = Objects.requireNonNull(defaultCountry);
+        this.defaultCountry = defaultCountry;
```

Comment:
Is defaultCountry still use?

Reply:
since this class has been deleted, this question is not relevant anymore i guess.